### ISSUE TITLE
ISO Date for Short Display Date and Long Display Date

### ISSUE THREAD

**Issue by jeffmcneill** [getgrav/grav#2283]
The drop-down list in Admin > Configuration > System for the "Short Display Date" and "Long Display Date" are rather limited. Because they are in a drop-down list there is no option to include an alternate.

Preferably, this drop-down could be replaced with a text field which would accept any valid PHP Date Format: https://secure.php.net/manual/en/datetime.formats.date.php

If not, or in the meantime, make available an ISO Short and ISO Long display date, such as:

YYYY-MM-DD
YYYY-MM-DD HH:MM

See also: https://en.wikipedia.org/wiki/ISO_8601

**Comment by jeffmcneill** [comment]
I agree one cannot include all valid options, but if there is a list one must choose from, then a few more can't hurt (much). If it were better to not have a list, or have a list with an open ended option. The way that Wordpress allows for about four choices or a fill-in-the-blank, with a link to documentation (and that eventually goes to the various options for PHP).

![image](https://user-images.githubusercontent.com/3119827/50054638-bf5fb500-0176-11e9-9d83-5d331d701a08.png)

**Comment by rhukster** [comment]
Added for Grav 2.1.0. The short and long date format pickers now include ISO 8601 presets, `Y-m-d` and `Y-m-d H:i`. The Grav 2 admin's date format field also has a **Custom…** option with a live preview, so any PHP date format can be typed in directly, which covers the free-text half of this request.
